accounts/tests.py

```diff
@@ -0,0 +1,59 @@
+from django.test import TestCase
+from django.conf import settings
+from django.contrib.auth import SESSION_KEY
+from django.contrib.auth import get_user_model
+from django.contrib.messages import get_messages
+from django.db.utils import IntegrityError
+from django.urls import resolve, reverse
+
+
+from .views import singup
```

Comment:
views.pyで作成したものはsignupではなくSignUpViewですので、
```suggestion
from .views import SignUpView
```
としてください。とはいえ、今回のテストで使用しているようには見えないのでそもそもimportしなくていいです。